src/StatusQ/Controls/StatusBaseInput.qml

```diff
@@ -0,0 +1,132 @@
+import QtQuick 2.14
+
+import StatusQ.Controls 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+
+Item {
+    id: statusBaseInput
+
+    property bool multiline: false
+
+    property bool clearable: false
+
+    property alias inputMethodHints: edit.inputMethodHints
+
+    property alias selectedText: edit.selectedText
```

Comment:
If you would split this control into TextField and TextArea, you could derive from them explicitly and customize the look and feel. In this case you don't need to alias all the properties.

Reply:
Yes, I was preferring having a `StatusBaseInput` and something like `StatusTextArea` as well. @borismelnik mentioned something about some `QDocument` or something that we'd need and we'll get access to with the text component we use in this PR.